### PR TITLE
[feat]: Add translator options to ElasticSearch and VSD queries

### PR DESCRIPTION
**Pull request by nxanil**
@natabal @ronakmshah 

You can now call the translator from the query template.
So the sample call will be like:
```
                        {
                            "term": {
                                "nsg":"{{nsg:call('demo1')}}"
                            }
                        },
```
Now you just need to pass the translator name inside **call**.

Then in the URL subsittue the value of respective context, i.e. **nsg** in above example like:
```
                       nsg=NSG1
```
By doing so you can pass the NSG1 to the called translator i.e. **demo** in above case.

Also don't forget to create your translator file at: _utils/translators_
and register is inside the _utils/translators/index.js_

Let me know if you have any issues in using this.

### CODE REVIEW

File: src/utils/helpers/json_templates.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
+/**
```
Comment (natabal) [inline src/utils/helpers/json_templates.js]: I think we need the copyright to be added if we modify this file. You could push the changes to the library's github. I think this is good functionality!

File: src/utils/configurations.js [hunk 2]
```diff
@@ -46,27 +70,29 @@ export const parameterizedConfiguration = (configuration, context) => {
...
         // else ignore the parameter because it is not used in the provided configuration.
     }
 
-    return queryParams;
+    return queryParams
 }
 
 export const contextualize = (data, context) => {
-    return parse(data)(context);
+    let template = parse(data)
```
Comment (natabal) [inline src/utils/configurations.js]: const template

File: src/utils/helpers/json_templates.js (hunk 1 above)
Comment (nxanil) [reply src/utils/helpers/json_templates.js]: Done, I will push it to library as well.

Comment (natabal) [reply src/utils/helpers/json_templates.js]: Thanks @nxanil let's push it to master for now and then once the lib is updated we can use the package

**Pull request merged**